### ISSUE TITLE
Cannot find module '@babel/runtime/helpers/interopRequireWildcard' on fresh install

### ISSUE THREAD

**Issue by bbuehrle** [symfony/webpack-encore#935]
I started with a fresh ```symfony new --full test``` install of Symfony, followed by ```composer require encore``` and ```yarn install``` (tried running with and without ```--force``` switch).  Upon running ```yarn dev``` and ```yarn build``` in either WSL2 (Ubuntu 20.04) or through PowerShell, I end up with this missing module:

```
yarn run v1.22.10
$ encore dev
Running webpack ...

 ERROR  Failed to compile with 1 errors                                                                       9:37:47 AM

 error  in ./assets/app.js

Syntax Error: Error: Cannot find module '<snip>\test\node_modules\@babel\runtime\helpers\interopRequireWildcard.js'


Entrypoint app 12.6 KiB = runtime.js 11.3 KiB app.js 1.32 KiB
webpack compiled with 1 error
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

It's been a while since I started a new project with Symfony, but I've never needed anything more than the basic commands to get started.  Did I miss a new setup step?  Wanting to play around with the new Symfony UX Stimulus. 😄

**Comment by stof** [comment]
can you share your package.json and your webpack.config.js ?

**Comment by bbuehrle** [comment]
I can still post if you want to look further, but after posting this issue I deleted the project folder.  When I just now rebuilt the project using the same steps I listed above, webpack compiled correctly.

Before I posted the issue, the folder ```node_modules\@babel\runtime\helpers\``` only contained 3-4 files and now there is 82, so it looks like the extraction process failed the first couple times I tried it this morning.
